spec/system/task_spec.rb

```diff
@@ -85,8 +78,7 @@
 
       it '既にステータスが完了のタスクのステータスを変更した場合、Taskの完了日が更新されないこと' do
         # TODO: FactoryBotのtraitを利用してください
-        project = FactoryBot.create(:project)
-        task = FactoryBot.create(:task, project_id: project.id, status: :done, completion_date: Time.current.yesterday)
+        task = create(:task, :finished_task, project_id: project.id)
```

Comment:
taskに対してのtraitなので`finished_task`の`_task`は省略しても良いかなと思います。

また代入するテストデータの意味合いが変わったので、対応する変数名は変えておきましょう。
```suggestion
        finished_task = create(:task, :finished)
```

Reply:
b9cb100 で対応しました！